### DISCUSSION TITLE
The UI doesn't seem to be connected to the same database

### DISCUSSION THREAD

**Discussion by WilliamBurdett** [PrefectHQ/prefect#3704] [Q&A] [answered]
I have a ubuntu server running prefect. And if I use the UI to access the database and do a query, it finds no flows. But if I go to the same IP address in my browser but instead the graphql port, it does find flows from my simple agent script.

http://<MY_IP_ADDRESS>:8080/default/api?request=
http://<MY_IP_ADDRESS>:4200/

There is nothing else installed or running on this server, and I have no idea how to fix it. It seems like there are 2 separate graphql databases, one for the UI and one for the agent code to connect to.

I'm using this query:
```
{
  flow {
    created,
    updated
  }
}
```

agent code:
```
import prefect
from prefect import task, Flow


@task
def welcome_logger(): 
    logger = prefect.context["logger"]
    logger.info("Welcome")
flow = Flow("Welcome Flow", tasks=[welcome_logger])
flow.register(project_name="Hello, World!") 
flow.run_agent()
```

**Comment by joshmeek** [comment, marked as answer]
Hi @WilliamBurdett the UI is rendered client side which will require you to change the address that your UI is looking at to the address of the Apollo endpoint running on your Ubuntu server: https://docs.prefect.io/orchestration/server/deploy-local.html#ui-configuration

**Reply by WilliamBurdett** [reply]
So I set the apollo_url in the config file and still have the same issue.

The UI says "Connected to Prefect Server at http://localhost:4200/graphql"

I tried setting the path to both localhost and my IP address. It seems to have the same problem regardless

**Reply by WilliamBurdett** [reply]
Okay, figured it out how to change in the UI. Thanks for the help!!
